moveit_ros/planning/trajectory_execution_manager/src/trajectory_execution_manager.cpp

```diff
@@ -171,6 +181,9 @@ void TrajectoryExecutionManager::initialize()
 
   reconfigure_impl_ = new DynamicReconfigureImpl(this);
 
+  if (!csm_)
+    ROS_ERROR_NAMED("traj_execution","Trajectory validation before execution is disabled, because no CurrentStateMonitor was provided.");
```

Comment:
This sentence was hard to read - remove the "before execution" part

Also, I think this should just be a warning